### PR TITLE
Updated path prefixes

### PR DESCRIPTION
**Pull request by ankur12-1610**
Updated the path prefixes according to the new flow: `Updater > Tabulator (turn a grid into the user-viewed grid) > Summarizer`

Fixes: #1123

### CODE REVIEW

File: pkg/api/v1/state.go
Comment (michelle192837) [inline pkg/api/v1/state.go:67]: nit: I think GroupGrid() shouldn't be referenced anymore with the lines below removed, so it should be safe to delete entirely. (It's good to confirm that's the case though).

**Review by michelle192837 — CHANGES_REQUESTED** [review]
Think this looks good, thanks! Left a comment.

In addition, last action before this is good to approve:
- Manually verify that none of the removed flags are referenced in files under https://github.com/GoogleCloudPlatform/testgrid/tree/master/cluster

File: pkg/api/v1/state.go
Comment (sultan-duisenbay) [reply pkg/api/v1/state.go:67]: +1 to this. This fetches the data from the updater rather than tabulator

**Review by sultan-duisenbay — CHANGES_REQUESTED** [review]
@ankur12-1610 this looks good to me. However, the tests are failing and I think I know why. since we removed the gridpathprefix, we are not looking in the grid path folder anymore and defaulting to tabs path prefix.

change the `serverTabPathPrefix` in the fake to "tabs"
in all the API test cases where the Grid is used, change the path to `gs://{bucket}/tabs/{dashboard_name}/{tab_name}` from `gs://{bucket}/grid/{testgroup_name}` and see if this helps

**Comment by ankur12-1610** [comment]
@michelle192837 @sultan-duisenbay fixed the tests, they are green now and have updated the code according to the reviews :)

Verified https://github.com/GoogleCloudPlatform/testgrid/tree/master/cluster as well, nothing to change.

**Comment by ankur12-1610** [comment]
Screenshot from 2023-02-25 12-46-41

These are the files that still have `GridPrefixPath`, I didn't remove because I thought it was related to the tabulator which requires `GridPrefixPath`.

**Comment by chases2** [comment]
> These are the files that still have `GridPrefixPath`, I didn't remove it because I thought it was related to the tabulator which requires `GridPrefixPath`.

That makes sense. The tabulator should have a configurable "grid" prefix (to read from) and "tabs" prefix (to write to).

Another way to think of it is that the "grid" is the output of the updater (to grab data) and "tabs" is the output of the tabulator (to refine data). They output the same ["shape"](https://github.com/GoogleCloudPlatform/testgrid/blob/master/pb/state/state.proto) of data, just different amounts of it.

**Comment by chases2** [comment]
Looks good to me, but I'll leave it to @michelle192837 (who filed the inital bug) to approve.

/lgtm
/lint

**Comment by ankur12-1610** [comment]
> > These are the files that still have `GridPrefixPath`, I didn't remove it because I thought it was related to the tabulator which requires `GridPrefixPath`.
> 
> That makes sense. The tabulator should have a configurable "grid" prefix (to read from) and "tabs" prefix (to write to).
> 
> Another way to think of it is that the "grid" is the output of the updater (to grab data) and "tabs" is the output of the tabulator (to refine data). They output the same ["shape"](https://github.com/GoogleCloudPlatform/testgrid/blob/master/pb/state/state.proto) of data, just different amounts of it.

Yeah got it. At the start, it was a little bit confusing about what to ignore and what not to, so to keep it simple I just ignored all the Tabulator-related occurrences :sweat_smile:

**Comment by sultan-duisenbay** [comment]
> @michelle192837 @sultan-duisenbay fixed the tests, they are green now and have updated the code according to the reviews :)
> 
> Verified https://github.com/GoogleCloudPlatform/testgrid/tree/master/cluster as well, nothing to change.

thanks @ankur12-1610 - LGTM!

**Review by sultan-duisenbay — APPROVED** [review]

**Pull request merged**
